### PR TITLE
Update for Twig 2.7: `Template` -> `TemplateWrapper`

### PR DESCRIPTION
**Pull request by bobdenotter**
Since Twig 2.7, `$twig->resolveTemplate($template)` doesn't return `Twig\Template` but `Twig\TemplateWrapper`. This PR simply updates the typehints accordingly.

### CODE REVIEW

**Comment by bobdenotter** [comment]
Also, I've updated the composer dependencies, because it wouldn't install otherwise since the vulnerability disclosure in Twig 2.6.

**Comment by EnriqueGF** [comment]
Thank's you!

File: composer.lock [hunk 1]
```diff
@@ -5045,16 +5169,16 @@
         },
         {
             "name": "twig/twig",
-            "version": "v2.6.2",
+            "version": "v2.7.2",
```
Comment (aurelijusb) [inline composer.lock]: :+1: Also waiting for [CVE-NONE-0001](https://symfony.com/blog/twig-sandbox-information-disclosure) fix to be merged to master.

**Comment by javiereguiluz** [comment]
Thanks for fixing this bug Bob.

**Comment by Spreeuw** [comment]
Has this bug been reintroduced with later versions of Twig? I have **2.8.1** and got the inverse error:

> Argument 1 passed to App\Twig\SourceCodeExtension::getTemplateSource() must be an instance of Twig\TemplateWrapper, instance of __TwigTemplate_162c098ca5b32414d11a6bd21c615e86daeea569ccab9c2c836dc6159b60cd1f given

Reverting the changes in `src/Twig/SourceCodeExtension.php` resolved the issue for me...

**Pull request closed without merging**